.github/workflows/check-release.yml

```diff
@@ -22,6 +22,8 @@ jobs:
       - name: Base Setup
         uses: jupyterlab/maintainer-tools/.github/actions/base-setup@v1
 
+      - run: python -m pip install -U setuptools pip
```

Comment:
Normally the `base-setup` action should already do the update here:

https://github.com/jupyterlab/maintainer-tools/blob/de48d520daf0fa754ebc139792dd879f55988763/.github/actions/base-setup/action.yml#L79